hrjobcontract/js/src/job-contract/directives/loader.js

```diff
@@ -9,48 +9,71 @@ define([
     $log.debug('Directive: hrjcLoader');
 
     return {
-      link: function ($scope, el, attrs) {
+      link: function ($scope, element, attrs) {
         var loader = document.createElement('div');
-        var loaderSet = false;
-        var positionSet = false;
+        var isLoaderSet = false;
+        var isPositionSet = false;
 
         loader.className = 'hrjc-loader spinner';
 
-        function isPositioned () {
-          var elPosition = window.getComputedStyle(el[0]).position;
-          return elPosition === 'relative' || elPosition === 'absolute' || elPosition === 'fixed';
-        }
+        (function init () {
+          if (attrs.hrjcLoaderShow) {
+            appendLoader();
+          }
 
+          initLIsteners();
+        }());
+
+        /**
+        * Appends the loader in the element and
+        * updates the element's position
+        */
         function appendLoader () {
           if (!isPositioned()) {
-            el.css('position', 'relative');
-            positionSet = true;
+            element.css('position', 'relative');
+            isPositionSet = true;
           }
 
-          el.append(loader);
-          loaderSet = true;
+          element.append(loader);
+          isLoaderSet = true;
         }
 
-        function removeLoader () {
-          loaderSet && loader.parentNode.removeChild(loader);
-          loaderSet = false;
+        /**
+         * Returns if the element has position value set
```

Comment:
`Checks if...` otherwise it sounds like it returns something if only the position value is set and does not return if the position is not set.